apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/startModal/NyIdent/NyIdent.js

```diff
@@ -69,67 +73,80 @@ export const NyIdent = ({ onAvbryt, onSubmit, zBruker }) => {
-					<div className="ny-ident-form_selects">
-						<FormikSelect
-							name="identtype"
-							label="Velg identtype"
-							size="medium"
-							options={Options('identtype')}
-							isClearable={false}
-						/>
-						<FormikTextInput name="antall" label="Antall" type="number" size="medium" />
-					</div>
-					<div className="ny-ident-form_maler">
-						<div>
-							<StyledH3>Opprett fra mal</StyledH3>
-							<DollyCheckbox
-								name="aktiver-maler"
-								onChange={() => handleMalChange(formikBag)}
-								label="Vis"
-								isSwitch
-							/>
-						</div>
+			{(formikBag) => {
+				const valgtMal = malOptions.find((mal) => mal.value === _get(formikBag.values, 'mal'))
+				const valgtMalTpsfValues = _get(valgtMal, 'data.bestilling.tpsf')
+				const erTpsfMal = tpsfAttributter.some((a) => _has(valgtMalTpsfValues, a))
 
-						<InputDiv>
-							<DollySelect
-								name="zIdent"
-								label="Bruker"
-								isLoading={state.loading}
-								options={zIdentOptions}
+				return (
+					<div className="ny-ident-form">
+						<h3>Velg type og antall</h3>
+						<div className="ny-ident-form_selects">
+							<FormikSelect
+								name="identtype"
+								label="Velg identtype"
 								size="medium"
-								onChange={(e) => handleBrukerChange(e, formikBag)}
-								value={zIdent}
+								options={Options('identtype')}
 								isClearable={false}
-								disabled={!malAktiv}
 							/>
-							<FormikSelect
-								name="mal"
-								label="Maler"
-								isLoading={state.loading}
-								options={malOptions}
-								size="grow"
-								fastfield={false}
-								disabled={!malAktiv}
-							/>
-						</InputDiv>
-						<div className="mal-admin">
-							<Button kind="maler" onClick={() => {}}>
-								<NavLink to="/minside">Administrer maler</NavLink>
-							</Button>
+							<FormikTextInput name="antall" label="Antall" type="number" size="medium" />
 						</div>
+						<div className="ny-ident-form_maler">
+							<div>
+								<StyledH3>Opprett fra mal</StyledH3>
+								<DollyCheckbox
+									name="aktiver-maler"
+									onChange={() => handleMalChange(formikBag)}
+									label="Vis"
+									isSwitch
+								/>
+							</div>
+
+							<InputDiv>
+								<DollySelect
+									name="zIdent"
+									label="Bruker"
+									isLoading={state.loading}
+									options={zIdentOptions}
+									size="medium"
+									onChange={(e) => handleBrukerChange(e, formikBag)}
+									value={zIdent}
+									isClearable={false}
+									disabled={!malAktiv}
+								/>
+								<FormikSelect
+									name="mal"
+									label="Maler"
+									isLoading={state.loading}
+									options={malOptions}
+									size="grow"
+									fastfield={false}
+									disabled={!malAktiv}
+								/>
+							</InputDiv>
+							{erTpsfMal && (
+								<Alertstripe type={'advarsel'} style={{ width: '97%' }}>
+									Denne malen utdatert, og vil dessverre ikke fungere som den skal. Dette fordi
```

Comment:
`Denne malen utdatert`, kanskje prøve å endre litt på den? 👼

Reply:
😈

Reply:
Det ble veldig mye bedre!

Reply:
Tusen takk for at du med ditt sylskarpe rettskrivningsblikk oppdaget dette! 🤩 🤩 🤩

Reply:
Tro meg, gleden var på min side! 🤓

Reply:
Godt å høre, er så viktig at du deler din kompetanse med oss som ikke er fullt så flinke på sånt! 🤩

Reply:
Øvelse gjør mester vettu, young padawan 💪